README.md

```diff
@@ -0,0 +1,149 @@
+
+> ~/.config/protonfixes/localfixes/default.py
+
+Note that local fixes override the included "global" fixes. So using an existing fix might be a good starting point for modifications, as they will not be executed.
+
+For example, overriding the default fix will disable [the parameter parsing for Steam games](#start-parameter-fixes). This can be restored by copying [the global default file](gamefixes-steam/default.py) and applying your changes accordingly.
+
+## Non-Steam games
+
+As we support platforms other than Steam, these fixes can also be used by other launchers, such as Lutris, Heroic, Legendary or Bottles.
+
+The current implementation requires entries in a special database that looks up the `UMU-ID`, which consists of the prefix `umu-` and in most cases the SteamID (if the game exists on that platform).
+
+https://github.com/Open-Wine-Components/umu-database
+
+## Config file
+
+There is a config file located at:
+
+> ~/.config/protonfixes/config.ini
+
+It's not widely used at the moment, but you can configure some aspects.
+
+### Defaults
+
+```toml
+[main]
+enable_checks = true
+enable_global_fixes = true
+
+[path]
+cache_dir = ~/.cache/protonfixes
+```
+
+### Variables
+
+`enable_checks`: De-/activate checks that are run on every startup. Currently only a check if esync might cause problems.
+
+`enable_global_fixes`: De-/activate global fixes, which are the built-in files in the `gamefix-*` folders. Only [local fixes](#local-fixes) will be executed.
+
+`cache_dir`: Used when something needs to be unpacked or similar.
+
+### Values 
+
+Valid "enable" values are: `yes`, `y`, `true`, `1`
+
+Any other value is interpreted as disabled.
+
+## Building binaries
+
+UMU-Protonfixes ships some binaries - used by some fixes - that need to be built.
+
+**You probably do not need to build them to contribute.**
+
+These binaries are not included in this repository, but are built from git-submodules in the [subprojects folder](subprojects/). To get the submodules, run the following command:
+
+```bash
+git submodule update --init --recursive
+```
+
+To actually build the binaries, simply run:
+
+```bash
+make
+```
+
+## Contributing
+
+We do enforce some linting and testing that can and should be done locally - before you open a pull request.
+
+### Linting
+
+To install the necessary tools, just run:
+
+```bash
+python -m pip install --upgrade pip
+pip install ruff
+```
+
+On Arch / Manjaro, you should run this instead:
+
+```bash
+sudo pacman -Sy ruff
+```
+
+To run a check, just execute:
+
+```bash
+ruff check .
+```
+
+You are good to go when you see the message `All checks passed!`.
+
+### Testing
+
+The filenames of the fixes are checked against the Steam and GOG APIs. All symbolic links are also checked. This is not mandatory to run locally, as you should have a working game fix to begin with, and it will be done automatically by Github's CI.
```

Comment:
Once we get more fixes related to other stores, it'd be nice to validate their store IDs against an API too. Or if there isn't an API, maybe query the game's store page if it includes the game's identifier in the URL like Steam's.

Reply:
Sure, I was just documenting the current state.

Loosely related, but I had some discussion about the other stores here: https://github.com/Open-Wine-Components/umu-database/issues/39